include/odp/api/spec/classification.h

```diff
@@ -605,6 +615,25 @@ typedef struct odp_pmr_param_t {
 
 } odp_pmr_param_t;
 
+/**
+ * Packet Matching Rule creation options
+ */
+typedef struct odp_pmr_create_opt_t {
+	/** Array of odp_pmr_param_t entries, one entry per term desired.
+	 * Use odp_cls_pmr_param_init() to initialize parameters into their default values.
+	 */
+	odp_pmr_param_t *terms;
+	/** Number of terms in the match rule. */
+	int num_terms;
+	/** The mark value to retrieve it through odp_packet_cls_mark() on PMR match.
```

Comment:
The grammar needs to be fixed. In addition, it is IMHO inaccurate to say that mark will be zero (or default) on PMR miss when in reality the mark in that case is not changed but may already have a non-default value as set by some other PMR.

I would rephrase the whole thing with something like this:
"Value to be set in the CLS mark of a packet when the packet matches this Packet Matching Rule. The default value is zero. The maximum value is indicated in 'max_mark' capability."

Reply:
Changed to :
        /** Value to be set in the CLS mark of a packet when the packet matches this
         * Packet Matching Rule.
         * The default value is zero.
         * The maximum value is indicated in odp_cls_capability_t::max_mark capability.
         */